docs/content/docs/connectors/pipeline-connectors/kafka.md

```diff
@@ -178,6 +185,63 @@ An output example is:
   }
 }
 ```
+When `sink.debezium-json.include-schema.enabled` is true, the output format will be:
```

Comment:
What about add one more example when `sink.debezium-json.include-schema.enabled` is false to help better comparison.

Reply:
The current [document ](https://nightlies.apache.org/flink/flink-cdc-docs-master/docs/connectors/pipeline-connectors/kafka/#debezium-json)already has examples where the default value is false